reflex/components/component.py

```diff
@@ -1007,6 +1007,14 @@ def _get_hooks_imports(self) -> imports.ImportDict:
             )
         return _imports
 
+    def add_imports(self) -> Dict[str, Union[str, List[str]]]:
```

Comment:
```suggestion
    def add_imports(self) -> Dict[str, Union[str, List[str], List[ImportVar]]]:
```

I think we should also allow the user to return a list of ImportVar as well, in case they need the more advanced functionality (like installation, but not rendering)

Reply:
Wouldn't it be better to have `List[Union[str, ImportVar]` then ?
If I want to use ImportVar for a specific tag I don't necessarily want to have to wrap all the tags in ImportVar.